pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: dart_style
 # Note: See tool/grind.dart for how to bump the version.
-version: 1.2.5-dev
+version: 1.2.5
 author: Dart Team <misc@dartlang.org>
 description: Opinionated, automatic Dart source code formatter.
```

Comment:
Mind pulling in the longer description? https://github.com/dart-lang/dart_style/pull/788/files#diff-fb2d52cd953afedcafc9fe7629c6f132R5

```
description: >-
  Opinionated, automatic Dart source code formatter.
  Provides an API and a CLI tool.
```

Reply:
Aye, captain!